packages/forms-web-app/src/lib/documents-api-wrapper.js

```diff
@@ -2,10 +2,62 @@ const fs = require('fs');
 const fetch = require('node-fetch');
 const FormData = require('form-data');
 const uuid = require('uuid');
+const { utils } = require('@pins/common');
 
 const config = require('../config');
 const parentLogger = require('./logger');
 
+async function handler(path, method = 'GET', opts = {}, headers = {}) {
+  const correlationId = uuid.v4();
+  const url = `${config.documents.url}${path}`;
+
+  const logger = parentLogger.child({
+    correlationId,
+    service: 'Documents Service API',
+  });
+
+  try {
+    logger.debug({ url, method, opts, headers }, 'New call');
+
+    return await utils.promiseTimeout(
+      config.appeals.timeout,
```

Comment:
do we want a specific documents service timeout that differs from the appeals timeout?

Reply:
I think keep the one. now poorly named, yes, but for now good enough.